core_api/src/routes/chat.py

```diff
@@ -25,12 +34,31 @@
 )
 
 
+# === LLM setup ===
+
+
 llm = ChatLiteLLM(
     model="gpt-3.5-turbo",  # TODO: set with env var
     # TODO: set max_tokens and temperature
     streaming=True,
 )
 
+es = env.elasticsearch_client()
+embedding_function = SentenceTransformerEmbeddings()
+# TODO: do we want to be able to set hybrid to True depending on Elastic subscription levels?
+# see: https://github.com/i-dot-ai/redbox-copilot-streamlit/blob/3d197e76e6d42cfe0b70f66bb374c899b74754b4/streamlit_app/utils.py#L316C17-L316C45
+hybrid = False
+
+# TODO: fix this - I don't think it's correctly implemented, as we're getting an error when making a request:
+# raise HTTP_EXCEPTIONS.get(meta.status, ApiError)(
+# elasticsearch.NotFoundError: NotFoundError(404, 'status_exception', '[file] is not an inference service model or a deployed ml model')
+vector_store = ElasticsearchStore(
+    es_connection=es,
+    index_name="redbox-data",
```

Comment:
```suggestion
    index_name="redbox-data-chunk",
```
maybe should be `file`? not sure tbh